templates/content/node--oe-sc-event--oe-w-content-banner.html.twig

```diff
@@ -4,11 +4,15 @@
  * Event content banner display.
  */
 #}
+{% set _content %}
+  {{ content.oe_summary }}
+  {{ content.oe_sc_event_registration_url }}
+{% endset %}
 <div{{attributes}}>
   {{ pattern('content_banner', {
     background: 'gray',
     title: label,
-    content: content.oe_summary,
+    content: _content,
```

Comment:
Note:
We _could_ inline this using render element composition:
```twig
  {{ pattern('content_banner', {
    background: 'gray',
    title: label,
    content: [
      content.oe_summary,
      content.oe_sc_event_registration_url,
    ],
    image: image,
  }) }}
```
But let's keep it as it is for now.